spec/index.bs

```diff
@@ -139,6 +142,33 @@ spec: WebIDL; urlPrefix: https://heycam.github.io/webidl/#
     </pre>
 
     In the above example <code>getMessagesFromOutbox</code> and <code>removeMessagesFromOutbox</code> are developer-defined functions.
+
+    Requesting a <a>periodic Background Sync opportunity</a> at a 60 second intervals from a <a>browsing context</a>:
+
+    <pre class="lang-js">
+      function getLatestNews() {
+        // Wait for the scoped service worker registration to get a
+        // service worker with an active state
+        return navigator.serviceWorker.ready;
+        .then(reg => {
```

Comment:
.then needs to be attached to ready

Reply:
Formatting is still a bit off. I would suggest something like this:
```js
navigator.serviceWorker.ready
    .then(reg => {
      return reg.periodicSync.register('get-latest-news', { minInterval: 60000 });
    })
    .then(() => {
      console.log('Periodic Sync registered!');
    })
    .catch(() => {
      console.log('Periodic Sync registered!');
    });
```

or this
```js
navigator.serviceWorker.ready
    .then(reg => reg.periodicSync.register('get-latest-news', { minInterval: 60000 }))
    .then(() => console.log('Periodic Sync registered!'))
    .catch(() => console.log('Periodic Sync registered!'));
```